data-prepper-plugins/otel-metrics-raw-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/otelmetrics/MetricsPluginSumTest.java

```diff
@@ -109,6 +110,63 @@ public void test() throws JsonProcessingException {
         assertSumProcessing(map);
     }
 
+    @Test
+    public void missingNameInvalidMetricTest() throws JsonProcessingException {
```

Comment:
It could be good to include a single valid metric to ensure that it processes that as well. I'll approve this as is, but feel free to consider a follow-on PR!

Reply:
Pushed a new diff with this change.